hack/test-go.sh

```diff
@@ -188,6 +188,7 @@ else
             -o -path "${kubernetes_path}/test/e2e"                                                    \
             -o -path "${kubernetes_path}/cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned" \
             -o -path "${kubernetes_path}/pkg/storage/etcd3"                                           \
+            -o -path "${kubernetes_path}/third_party/golang/go/build"                                 \
```

Comment:
yes, we can disable i think

On Fri, May 20, 2016 at 10:53 AM, Michal Fojtik <notifications@github.com>
wrote:

> In hack/test-go.sh
> <https://github.com/openshift/origin/pull/8856#discussion_r64054032>:
>
> > @@ -188,6 +188,7 @@ else
> >              -o -path "${kubernetes_path}/test/e2e"                                                    \
> >              -o -path "${kubernetes_path}/cmd/libs/go2idl/client-gen/testoutput/testgroup/unversioned" \
> >              -o -path "${kubernetes_path}/pkg/storage/etcd3"                                           \
> > +            -o -path "${kubernetes_path}/third_party/golang/go/build"                                 \
>
> @smarterclayton <https://github.com/smarterclayton> this is failing with:
>
> # github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/third_party/golang/go/build
> package github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/third_party/golang/go/build (test)
>     imports internal/testenv: use of internal package not allowed
> FAIL    github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/third_party/golang/go/build [setup failed]
>
> ok to disable?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8856/files/7615475893b4f0759006debda6c9fe38fc843efc..e0be29ba8e61ede54b39191d860dce28517eb3f8#r64054032>
>